src/components/charts/stacked-area/stacked-area.js

```diff
@@ -155,7 +160,8 @@ class ChartStackedArea extends PureComponent {
                 (
                   <CustomYAxisTick
                     customstrokeWidth="0"
-                    unit="t"
+                    unit={unit}
```

Comment:
You're passing `unit` here but you removed it from props in `CustomYAxisTick` and not using it anywhere